<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMC</class>
 <widget class="QMainWindow" name="IMC">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>274</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>582</width>
    <height>274</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>582</width>
    <height>274</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IMC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>97</width>
        <height>211</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>Datos</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Fecha</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QDateEdit" name="inFecha">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2022</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Altura</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="inAltura">
         <property name="maximum">
          <double>399.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>8.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Peso</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QDoubleSpinBox" name="inPeso">
         <property name="maximum">
          <double>500.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>kg</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="btnCalcular">
         <property name="text">
          <string>Calcular</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>461</width>
        <height>211</height>
       </size>
      </property>
      <property name="title">
       <string>Resultados</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>23</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Peso Actual</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>149</x>
         <y>23</y>
         <width>42</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Peso min</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>23</y>
         <width>46</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Peso max</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="outPesoActual">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>42</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="outPesoMin">
       <property name="geometry">
        <rect>
         <x>149</x>
         <y>42</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="outPesoMax">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>42</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Altura</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="outAltura">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>IMC</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="outResultado">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>139</x>
         <y>81</y>
         <width>281</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
